README.md

```diff
@@ -142,30 +148,30 @@ By the end of this lesson. You should be able to set up two separate servers tha
 * **Question:** Describe what is happening in the code above.
 
 * **Your Answer:**
-
+local storage will retain the token even if you refresh or whatnot.
 ---
 
-- [ ] Now when you refresh the page, it looks as though you are logged in. Next, try clicking the logout button.
+- [x] Now when you refresh the page, it looks as though you are logged in. Next, try clicking the logout button.
 
 * **Question:** When you click "Logout", nothing happens unless you refresh the page. Why not?
 
 * **Your Answer:**
-
+Nothing happens visually, in the console, i see that the token is cleared from local storage. I think this happens because the app doesn't know to go and check back to see if it still has the user's token. We probably need to add an event listener.
```

Comment:
React automatically re renders when state changes, we have updated the local storage but not the state!